### PR TITLE
First version of launch files. 

### PR DESCRIPTION
**Pull request by griswaldbrooks**
Probably doesn't do anything.

### CODE REVIEW

**Comment by griswaldbrooks** [comment]
Note, these are copy/pasted from here: https://github.com/cra-ros-pkg/robot_localization

File: params/dual_ekf_navsat.yaml [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+    debug: false
+    
+    map_frame: map
+    odom_frame: odom
+    base_link_frame: base_link
+    world_frame: map
+    
+    odom0: odometry/wheel
+    odom0_config: [false, false, false,
+    false, false, false,
```
Comment (markwomack) [inline params/dual_ekf_navsat.yaml]: nit: indenting?

File: params/dual_ekf_navsat.yaml [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+    false, false, true,
+    false, false, false]
+    odom0_queue_size: 10
+    odom0_nodelay: true
+    odom0_differential: false
+    odom0_relative: false
+    
+    odom1: odometry/gps
+    odom1_config: [true,  true,  false,
+    false, false, false,
```
Comment (markwomack) [inline params/dual_ekf_navsat.yaml]: nit: indenting?

File: params/dual_ekf_navsat.yaml [hunk 3]
```diff
@@ -0,0 +1,165 @@
...
+    false, false, false]
+    odom1_queue_size: 10
+    odom1_nodelay: true
+    odom1_differential: false
+    odom1_relative: false
+    
+    imu0: imu/data
+    imu0_config: [false, false, false,
+    true,  true,  false,
```
Comment (markwomack) [inline params/dual_ekf_navsat.yaml]: nit: indenting?

**Comment by mwimble** [comment]
What is the nit? I’m not sure from the e-mail. YAML uses indention to express sub-structure.

**Comment by griswaldbrooks** [comment]
Yup. Indenting. I'll fix it here. Already fixed on what I'm working on.

**Comment by markwomack** [comment]
"nit" is used to indicate that it is not a huge deal, usually related to text formatting.

**Comment by markwomack** [comment]
As opposed to "ni" which is used to annoy people.

**Comment by griswaldbrooks** [comment]
I should change my handle to **nit**

**Comment by markwomack** [comment]
You are related to text formatting?

**Comment by griswaldbrooks** [comment]
Yup. My cousin is a linter.

**Review by markwomack — APPROVED** [review]

**Pull request merged**